planex/cmd/createmockconfig.py

```diff
@@ -53,16 +93,20 @@ def load_mock_reference(fname):
     return config_opts
 
 
-def load_yum_repos(includes, excludes):
+def load_yum_repos(repo_config_list):
     """
     read in the yum repository configuration
     """
     yum_base = yum.YumBase()
-    enabled_set = {repo for pattern in includes
-                   for repo in yum_base.repos.findRepos(pattern)}
-    disabled_set = {repo for pattern in excludes
-                    for repo in yum_base.repos.findRepos(pattern)}
-    return list(enabled_set - disabled_set)
+    repo_set = {repo for repo in yum_base.repos.findRepos('*')
```

Comment:
repo_set = set(repo ...

Reply:
@srowe This is set comprehension and will return a `set`, so we don't need to use a `set(...)` .

Reply:
Thanks, I hadn't come across it.

Reply:
I think you can replace this with:
```
set(yum_base.repos.listEnabled())
```